test/steps_test.go

```diff
@@ -1132,7 +1118,7 @@ func signBothEqual() error {
 
 func signMsigKmd() error {
 	kcl.ImportMultisig(handle, msig.Version, msig.Threshold, msig.Pks)
-	decoded, err := base32.StdEncoding.WithPadding(base32.NoPadding).DecodeString(pk)
+	decoded, _ := base32.StdEncoding.WithPadding(base32.NoPadding).DecodeString(pk)
```

Comment:
Is it better to ignore the `err` or explicitly add a check for the error here?

Reply:
Good point. I'll add the check and see if any steps break.

Reply:
Tests are passing after I percolated the error up. Resolving.